src/services/CLIService.ts

```diff
@@ -101,24 +114,42 @@ export class CLIService implements IService{
             userCompose,
             userComposeDir,
             blocklisting,
-            startup
+            startup,
+            verboseLevel
         };
 
         return currentArgv;
     }
 
     public setCurrentArgv(argv: ArgumentsCamelCase<{}>): void {
-        this.currentArgv = argv;
+        this.currentArgv = {
+            ...argv,
+            detached: this.defIsStartup(argv)
+        };
     }
 
-    private accountOption(yargs: Argv<{}>): void {
+    private defIsStartup(argv: yargs.ArgumentsCamelCase<{}>): boolean {
+        switch (argv._[0]) {
+            case 'stop':
+            case 'generate-accounts':
+            case 'debug':
+                return true;
+            case 'start':
+            case 'restart':
+                return false;
+            default:
+                return true;    
+        };
+    }
+
+    private static accountOption(yargs: Argv<{}>): void {
         yargs.positional('accounts', {
             describe: 'Generated accounts of each type.',
             default: 10
         });
     }
 
-    private detachedOption(yargs: Argv<{}>): void {
+    private static detachedOption(yargs: Argv<{}>): void {
```

Comment:
There is a need. 
Basically I found that the `middleware` of `yargs` is firing after the command definition handler. 
This would make me instantiate the `cliService` in the handlers themselves. 
But why actually ? These configuration functions. do not need a reference to an instance of the `cliService` class.
(no reference to the this object) In OOP terms they should be **static** in my opinion. 
And most importantly that `fixes` our instantiation problem.

Reply:
True, they make more sense in the context of static, leave them 👍